### PR TITLE
SimplyORM

### PR DESCRIPTION
**Pull request by vrtsev**
This PR contains some cucumber tests and metaprogramming homework - own ORM.

### CODE REVIEW

File: lib/simply_orm.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
+class SimplyORM
+  def initialize(host='localhost', user='blog', password='vadimblog', dbname='blog_development')
+    @host     = host
+    @user     = user
+    @password = password
+    @dbname   = dbname
```
Comment (petrokoriakin) [inline lib/simply_orm.rb]: It is better to read this values from config/database.yml

**Review by petrokoriakin — CHANGES_REQUESTED** [review]
It looks like the new shiny orm is unable to write data into database. Ability to update existing data and add new blog posts is essential for orm. And also to make sure your solution works, it should be integrated into the app.
My suggestions here are:
- write more cucumber tests to cover at least 80% of blog functionality
- enhance ORM to write data to database
- replace ActiveRecord with SimpleORM and deploy changes live.

File: lib/simply_orm.rb (hunk 1 above)
Comment (vrtsev) [inline lib/simply_orm.rb]: Thanks for the advice. I have a few questions:
1. What can you say about the structure of the class? Do I properly separated class into private and public methods?
2. In the code, I have had a lot of code duplication when I make new methods. How can I make refactoring for methods "define_method_all, ..find, ..where", to fit them all in one method?

**Comment by vrtsev** [comment]
Thanks for the advice. I have a few questions:

1. What can you say about the structure of the class? Do I properly separated class into private and public methods?
2. In the code, I have had a lot of code duplication when I make new methods. How can I make refactoring for methods "define_method_all, ..find, ..where", to fit them all in one method?

**Pull request closed without merging**